app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= t "posts.edit" %>
+<%= form_for :post, url: post_path(@post), method: :patch do |f| %>
+  <% if @post.errors.any? %>
+    <% @post.errors.full_messages.each do |msg| %>
+      <div class ="alert alert-danger">
+        <%= msg %>
+      </div>
+    <% end %>
+  <% end %>
```

Comment:
You should create file '_error_messages.html.erb', then move code lines to that file, then call back here